### PR TITLE
Deleting Note

### LINKED ISSUE #2: Create note deletion page UI

### PR DESCRIPTION
**Pull request by naufil50**
Fixes #2 

note-delete

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx [hunk 1]
```diff
@@ -1,40 +1,31 @@
...
+      Toastr.success("Note has been successfully deleted.");
     } catch (error) {
       logger.error(error);
-      setDeleting(false);
     }
   };
 
+  const closeAlert = () => {
+    setOpen(false);
+    onClose;
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx:19]: What's the point of writing this function here without invoking it?

**Review by yedhink — COMMENTED** [review]
@naufil50 One comment added.

File: app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx (hunk 1 above)
Comment (naufil50) [reply app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx:19]: Done, Thanks.

Comment (yedhink) [reply app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx:19]: @naufil50 Thanks for letting me know that you've made the change. But here a text message was unnecessary. Please read: https://handbook.neetokb.com/articles/acknowledging-a-message

**Pull request merged**